subprojects/platform-play/platform-play.gradle

```diff
@@ -28,3 +41,15 @@ useTestFixtures(project: ':platformBase')
 
 useClassycle()
 strictCompile()
+
+task integTestPrepare(type: org.gradle.testing.IntegrationTest) {
```

Comment:
Would it be simpler to just use an IntegrationTest with the existing source set and exclude/include it as necessary?

Adding another source set will add all of the other check tasks too (checkstyle, codenarc, etc)